### PR TITLE
Do not include passphrase with invites to published libraries

### PR DESCRIPTION
**Pull request by leogrim**
@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala [hunk 1]
```diff
@@ -12,7 +12,7 @@ import com.keepit.common.mail.template.EmailToSend
 import com.keepit.common.mail.template.TemplateOptions._
 import com.keepit.common.social.BasicUserRepo
 import com.keepit.common.mail.template.helpers.fullName
-import com.keepit.model.{ UserEmailAddressRepo, UserRepo, NotificationCategory, User, LibraryInvite, LibraryRepo, KeepRepo }
+import com.keepit.model._
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: IntelliJ?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: It was up to 8 imports. Can that be okay? It makes for massive headers, and especially for `com.keepit.model`, it's very clear what you're using.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: we need a consensus on this ...

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: Fight!

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: My other pull request brought back the list. I'll let you fight though.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:15]: not sure if this works:

http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

**Pull request merged**